container_images/tox/README.md

```diff
@@ -0,0 +1,63 @@
+A prow worker that executes Python tests using [tox](https://tox.readthedocs.io/).
+Check the Dockerfile for which Python versions are installed.
+
+### Interface
+
+Your job:
+1. Create a [tox configuration file](https://tox.readthedocs.io/en/latest/config.html) in your
+repository's test directory.
+   * Optionally, write test artifacts to the `ARTIFACTS` directory.
```

Comment:
does this mean a literal directory named ARTIFACTS ? is it a subdir of test/ ?

Reply:
if you mean $ARTIFACTS i follow, but you should add the dollar sign to make it clear. maybe 'optionally, configure your tests to write their output to the $ARTIFACTS directory, which will exist during test execution'

but below you've given an example where pytest is writing to $ARTIFACTS. what does this bullet point mean for the user? do they need to do anything to their tests?

Reply:
Re-worded to: Optionally, write test artifacts to the directory set in the `ARTIFACTS` environment variable.